addon/components/frost-tab.js

```diff
@@ -28,7 +28,7 @@ export default Component.extend(PropTypesMixin, {
     hook: PropTypes.string,
     parentHook: PropTypes.string,
     targetOutlet: PropTypes.string.isRequired,
-    selectedTab: PropTypes.string.isRequired,
+    selectedTab: PropTypes.string,
```

Comment:
There is was a test already in this repo that ensures no tabs are selected when this isn't present. Rather than deleting that test I just removed the `.isRequired` since I guess someone wanted that use case.